### DISCUSSION TITLE
Automatic Restart Timers

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#94] [Suggestions]
Would it be possible to add increments of 15 minutes to the options for restart times? With a huge cluster of machines I would love the ability to do a rolling restart, where we can have them restart at 15 minute intervals from each other. Thank you for your consideration. :)

**Comment by birdman3131** [comment]
![image](https://user-images.githubusercontent.com/749937/155281548-0c10731e-8246-49ce-854b-b29ec45c6d25.png)
This is doable by setting a scheduled task as shown in the image. I have 3 tasks. 1 set to a 30 minute countdown. 1 set to a delay of 10 minutes then a 30 minute countdown and then one set to 20 minutes and then a 30 minute countdown. I then set each task to run on different servers.

**Comment by mvaessen** [comment]
@elusive-ttv to be sure, you're referring to these right?
![Screenshot 2022-02-23 at 09 58 09](https://user-images.githubusercontent.com/13402423/155287962-87a79c33-94dd-47d2-ad64-fea9f020ac87.png)

**Reply by elusive-ttv** [reply]
Yes that's it!!

**Comment by mvaessen** [comment]
I've created a development item based on your suggestion #96 and hope to have this included in the next release. 
Thanks for your suggestion!
